patches/tModLoader/Terraria/Utils.cs.patch

```diff
@@ -62,7 +62,7 @@
 +		*/
 +
 +		Vector2 vector = ChatManager.GetStringSize(value, text, Vector2.One);
-+		ChatManager.DrawColorCodedStringWithShadow(spriteBatch, value, text, pos, color, 0f, new Vector2(anchorx, anchory) * vector, new Vector2(scale), -1f, 2f);
++		ChatManager.DrawColorCodedStringWithShadow(spriteBatch, value, text, pos, color, 0f, new Vector2(anchorx, anchory) * vector, new Vector2(scale), -1f, 1.5f);
```

Comment:
Based on the commented code above, it looks like the `spread` parameter here should be `1f`

Reply:
I think I did try 1f at first, but the border looked way too thin. When I used hot reload to compare vanilla to modified code, 1.5f seemed to have no change. I still have no clue why it is like this.

Reply:
given that this is a pixel offset parameter for the shadow text in the 4 axial directions, and Terraria only uses `NEAREST` filtering, sub-pixel values have no effect. So it should be either `1` or `2`

Reply:
Vanilla
![vanilla](https://user-images.githubusercontent.com/62684035/228401325-7ba4caf0-888b-42af-bc9f-e804329f8dc7.png)

Spread 1.5f
![1 5 spread](https://user-images.githubusercontent.com/62684035/228401471-cf5fe882-b756-41d9-831d-a2e0824396aa.png)

Spread 2f
![spread 2](https://user-images.githubusercontent.com/62684035/228401316-3d177d9f-28ea-44e4-8b53-9852e3b8026c.png)

Spread 1f
![1 spread](https://user-images.githubusercontent.com/62684035/228401322-b8e7c4a0-878b-4c82-8687-9e7e5e154fc6.png)